### DISCUSSION TITLE
Achieve Faster Inference Speeds with Ultralytics YOLOv8 & Intel’s OpenVINO

### DISCUSSION THREAD

**Discussion by pderrenger** [ultralytics/ultralytics#8152] [Show and tell]
Ultralytics YOLOv8 and [**Intel's OpenVINO™**](https://docs.ultralytics.com/integrations/openvino/) come together to triple inference speeds across CPUs and GPUs. Ideal for enhancing video analytics, smart cities, and retail, our latest blog provides a deep dive into leveraging this integration for optimal AI model performance.

🔍 **Key Highlights:**

- **Enhanced AI Performance:** Boost inference speeds by up to 3x, enabling faster, more efficient AI deployments.
- **Broad Hardware Compatibility:** Achieve seamless deployment across Intel's vast hardware ecosystem, ensuring your AI solutions are versatile and scalable.
- **Step-by-Step Optimization Guide:** Our blog walks you through the process of exporting and optimizing your YOLOv8 models for OpenVINO™, simplifying the path to high-performance AI applications.

Dive into our latest blog post for a step-by-step on harnessing the full potential of YOLOv8 with OpenVINO™!

**Learn More** 👉 https://www.ultralytics.com/blog/achieve-faster-inference-speeds-ultralytics-yolov8-openvino

[![OpenVINO (1)](https://github.com/ultralytics/ultralytics/assets/107626595/a7b72b67-4cd1-47a1-832d-b17a2a38c185)](https://www.ultralytics.com/blog/achieve-faster-inference-speeds-ultralytics-yolov8-openvino)

**Comment by glenn-jocher** [comment]
If you are running on Intel hardware, OpenVINO is unbeatable in terms of speed for YOLOv8 models, it's the best choice for Intel CPU inference.

**Comment by Harly-1506** [comment]
Hello everyone,

Check out my new GitHub repository for running YOLOv8 object detection and segmentation inference using OpenVINO and NumPy only. This implementation is faster than the Torch version, offering improved performance and efficiency. Visit the repository here: [Faster Inference YOLOv8](https://github.com/Harly-1506/Faster-Inference-yolov8).

Your feedback and contributions are welcome!

Thanks

**Reply by pderrenger** [reply]
Hello,

Great job on implementing YOLOv8 with OpenVINO for enhanced inference speed! 🚀 It's exciting to see community-driven projects that optimize performance. I'll definitely check out your repository.

For those interested in similar enhancements, remember that leveraging OpenVINO can significantly boost inference speeds, especially on Intel hardware. Here’s a simple example of how you might export a YOLOv8 model to OpenVINO format:

```python
from ultralytics import YOLO

# Load a model
model = YOLO('yolov8n.pt')

# Export to OpenVINO
model.export(format='openvino')
```

Keep up the great work, and I look forward to seeing how your project evolves!
